docs/block-edit-save.md

```diff
@@ -0,0 +1,107 @@
+# Edit and Save
+
+When registering a block, the `edit` and `save` functions provide the interface for how a block is going to be rendered within the editor, how it will operate and be manipulated, and how it will be saved.
+
+## Edit
+
+The `edit` function describes the structure of your block in the context of the editor. This represents what the editor will render when the block is used.
+
+```js
+// Defining the edit interface
+edit() {
+	return <hr />;
+}
+```
+
+This function can return properties through an object argument.
+
+### attributes
+
+This property returns all the available attributes and their values, as described by the `attributes` property when the block type was registered. In this case, assuming we had defined an attribute of `content` during block registration, we would receive and use that value in our edit function:
```

Comment:
Another odd use of "returns". I might expect "makes available", "exposes", "lists", "contains".